### DISCUSSION TITLE
Remove the margin in react-json-tree

### DISCUSSION THREAD

**Discussion by stoplion** [reduxjs/redux-devtools#1322] [Q&A]
There is top, left, and bottom margin hard coded to the react-json-tree root element. 
Is there any way to remove that?

**Comment by Methuselah96** [comment]
Yes, take a look at [advanced configuration](https://github.com/reduxjs/redux-devtools/tree/main/packages/react-json-tree#advanced-customization). You would want to override the [`tree` styling](https://github.com/reduxjs/redux-devtools/blob/main/packages/react-json-tree/src/createStylingFromTheme.ts#L48).

**Reply by stoplion** [reply]
thank you
